docs/release_notes/v0.10.0.md

```diff
@@ -0,0 +1,457 @@
+  
+# Dapr 0.10.0
+
+We're happy to announce the release of Dapr 0.10.0!
+
+As Dapr is approaching a 1.0 release, this release focuses on operational concerns in production environments, stability improvements, test infrastructure enhancements and as always, community requests and user feedback in the form of new features and bug fixes.
```

Comment:
This is probably longer trend than just one release, no? How about this 

> With Dapr approach v1.0 release, our releases increasingly focus on operational concerns like stability, reliability, and upgradability in production environments.

Reply:
Changed